data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/paths/_teku_v1_admin_add_peer.json

```diff
@@ -0,0 +1,46 @@
+{
+  "post" : {
+    "tags" : [ "Teku" ],
+    "operationId" : "AddPeer",
+    "summary" : "Add a peer to the node",
+    "description" : "Adds a peer to the node.",
+    "requestBody" : {
+      "content" : {
+        "application/json" : {
+          "schema" : {
+            "type" : "array",
+            "items" : {
+              "type" : "string"
+            }
```

Comment:
given we take an array, `add_peers` is probably a better name

Reply:
fixed